build.sh

```diff
@@ -72,25 +81,32 @@ function check_ut_control_revision()
 
 # Check if the common document configuration is present, if not clone it
 pushd ${FRAMEWORK_DIR} > /dev/null
+
+configure_ut_control() {
+    cd ./ut-control
+    #git checkout ${UT_CONTROL_PROJECT_VERSION}
+    git checkout feature/gh38-separate-obj-dir-ut-control
+    TARGET=${TARGET} ./configure.sh
+}
+
 if [ -d "${UT_CONTROL_LIB_DIR}" ]; then
     echo "Framework ut-control already exists"
     # ut-control exists so run the makefile from ut
     check_ut_control_revision
-    #make test
+    if [ -d "${CURL_OBJ_DIR}" ]; then
```

Comment:
why is this checkhere? shouldn't it be in confure_ut_control if anywhere? is it still required.

Reply:
I have renamed this to THIRD_PARTY_LIB_DIR, so that its clears the confusion.